<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Export Item to Excel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Item to Export</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="exportcomboBox">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Input Table</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Export Final Output</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Initial Results Table</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Observation Matrix Table</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Absolute Terms</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Most Probable Height</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Residuals</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Unit Variance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>99 per Confidence Level</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Path                                                                                             </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="path"/>
     </item>
     <item>
      <widget class="QPushButton" name="openpath">
       <property name="text">
        <string>open</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignRight">
    <widget class="QPushButton" name="okbtn">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
